recipes/pytest-notebook/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pytest-notebook" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c6117a4ca0ec404ac31c28a4baa21b4a3031f0a6d635efee023d46b20de15be3"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py < 35]
```

Comment:
Instead of the skip, you can put `python >=3.5` to host and run requirements and then make this `noarch: python` here.

Reply:
ah fair, I thought for some reason that you couldn't use `noarch` with python version restrictions. Fixed